motion/p5js/commonfateplanets/index.js

```diff
@@ -0,0 +1,148 @@
+// Kreis1
+var circleX;
+var circleY;
+var sizeX;
+var sizeY;
+
+// Kreis2
+var circleX2;
+var circleY2;
+var sizeX2;
+var sizeY2;
+
+// Kreis3
+var circleX3;
+var circleY3;
+
+//Kreis4
+var circleX4;
+var circleY4;
+
+//Kreis5
+var circleX5
+var circleY5
+
+
+function setup() {
+  createCanvas (500,500);
```

Comment:
the canvas needs to assigned to the sketch id div. see https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code